### DISCUSSION TITLE
Scatter plots

### DISCUSSION THREAD

**Discussion by MaxHalford** [excalidraw/excalidraw#2823] [Q&A]
Hey there. The copy/paste chart feature has blown my mind. As a data scientist, it's a game changer to my charting routine. Have you considering adding support for scatter plots? I like your philosophy of not doing too many things and doing them well. In the data world there is a holy trinity of scatter/line/bar charts that works for 90% of usecases, so it would seem appropriate to consider having those 3.

Peace out. Would love to help if necessary.

**Comment by dwelle** [comment]
I think we could do that. Would you be able to send a PR? :)

**Comment by lipis** [comment]
How does scatter plots work? We need two values for each point right? Because at the moment we support at least one column with numeric values.

**Reply by hydrosquall** [reply]
A classic scatter plot could scale up to 4 columns (x, y, radius, and color), but 2 would be the minimum (x and y). One could get fancier (bind shape, opacity, border color, border thickness) to additional columns, but at that point you might be looking for a dedicated charting tool.
